docs-src/concepts/what-is-a-schedule.md

```diff
@@ -183,6 +183,16 @@ And not C, even though C completed after A, because the result for D is captured
 
 Failures and timeouts do not affect the last completion result.
 
+:::note
+
+When a Schedule triggers a Workflow that completes successfully and yields a result, this result from the initial Schedule execution can be accessed by the subsequent scheduled execution through the `LastCompletionResult`.
```

Comment:
```suggestion
When a Schedule triggers a Workflow that completes successfully and yields a result, the result from the initial Schedule execution can be accessed by the subsequent scheduled execution through `LastCompletionResult`.
```
Per https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements.